nab/detectors/numenta/numentaTM_detector.py

```diff
@@ -25,7 +25,10 @@
 from nupic.algorithms import anomaly_likelihood
 from nupic.frameworks.opf.common_models.cluster_params import (
   getScalarMetricWithTimeOfDayAnomalyParams)
-from nupic.frameworks.opf.modelfactory import ModelFactory
+try:
+  from nupic.frameworks.opf.modelfactory import ModelFactory
+except:  # Try importing it the old way (version < 0.7.0.dev0)
```

Comment:
`modelfactory` is the old way, `model_factory` is the new way.
Also, [avoid inline comments](http://nupic.docs.numenta.org/contributing/python-style-guide.html#inline-comments).